charts/devnet/templates/chain/genesis.yaml

```
@@ -243,6 +244,10 @@ spec:
               port: 8000
             initialDelaySeconds: 10
             periodSeconds: 10
+      imagePullSecrets:
+      {{- range $imagePullSecrets }}
+        - name: {{ .name }}
+      {{- end }}
```

could we just have this as an optional param fully? based on if full chain has imagePullSecret.
Something like:

```suggestion
      {{-if $imagePullSecrets }}
      imagePullSecrets:
      {{- range $imagePullSecrets }}
        - name: {{ .name }}
      {{- end }}
      {{- end }}
```

---

Basically to avoid empty `imagePullSecrets` in the template.